lib/test/regression.js

```diff
@@ -6,37 +6,39 @@ const Crypt = require('../crypt.js');
 const Messenger = require('../messenger.js');
 const Settings = require('../settings.js');
 const path = require('path');
+const oldCryptFilename = Crypt.filename;
+const oldCryptFilepath = Crypt.filepath;
 
 describe('Crypt', () => {
-    it('getInstance() should create a new singleton object', () => {
-        const crypt = Crypt.getInstance();
```

Comment:
I had an issue using `getInstance`, so instead this PR monkey patches `Crypt`'s `filename` and `filepath` for testing.

It creates a `.test.fbmessenger.enc` file in the `lib/test` directory.